src/components/DateInput/TimePicker.tsx

```diff
@@ -49,6 +64,11 @@ const TimePicker: React.FC<TimePickerProps> = ({ date, onTimeUpdate, label = ''
 
   return (
     <>
+      {helperText && !is12Hours && (
```

Comment:
Perhaps this should be `as="label"` and (potentially) be related to the `Combobox` next  to it?

Perhaps an easy way to do this would be:

```jsx
<Box as="label">
   From Time
   <Box ml={2}>
       <Combobox />
   </Box>
</Box>
```

This way, clicking on the "From Time" would put focus on the Combobox's internal input. Does this feel like a hack?

Reply:
Nope, that's an amazing catch. I used some plain old `id` and `htmlFor` as wrapping everything with a Box messes the layout with the Flex wrapper.

Reply:
sure!